build_library/prod_image_util.sh

```diff
@@ -108,7 +108,8 @@ L+  /etc/ld.so.conf     -   -   -   -   ../usr/lib/ld.so.conf
 EOF
 
   # Move the PAM configuration into /usr
-  sudo mv ${root_fs_dir}/etc/pam.d ${root_fs_dir}/usr/lib
+  sudo mkdir -p ${root_fs_dir}/usr/lib/pam.d
+  sudo mv -n ${root_fs_dir}/etc/pam.d/* ${root_fs_dir}/usr/lib/pam.d/
```

Comment:
The directory isn't and shouldn't be empty following the new baselayout version. I tossed in -n on the off chance that something weird happened. What I really want is "fail if you would have clobbered" but there isn't any such option.